site/src/routes/formatting.ts

```diff
@@ -21,7 +29,7 @@ export function formatClasstime(time: Classtime): string {
     const endTime: string = endTimeArray[0] + ':'
         + formatMinutes(endTimeArray[1])
         + formatAmPm(endTimeArray[2]);
-    return days + ' ' + startTime + ' - ' + endTime;
+    return ' ' + startTime + ' - ' + endTime;
```

Comment:
Do you still want a space here?

Reply:
No, I'll get rid of it in a follow up PR. Thanks for the catch